validate_config.py can no longer be run standalone

During a pristine install on Qubes 2.4.2, I cannot complete a successful `make staging` run. What I did was:

1. Install Qubes according to https://workstation.securedrop.org/en/stable/admin/install/prepare.html#install-qubes-os-estimated-wait-time-30-45-minutes (no Whonix)
2. Run all updates
3. Clone the securedrop-workstation repo in the work VM, then send to dom0
4. Add config.json and sd-journalist.sec to the securedrop-workstation directory in dom0
5. `cd securedrop-workstation && make staging`

It fails at:

```
Traceback (most recent call last):
  File "/home/user/securedrop-workstation/./files/validate_config.py", line 21, in <module>
    from sdw_util.config_types import Dom0Config, ValidationError
ModuleNotFoundError: No module named 'sdw_util.config_types'
```

Full output:

<details>

```
[user@dom0 securedrop-workstation]$ make staging
gpg-pubkey-3fab65ab-660f2beb
gpg-pubkey-3fab65ab-660f2beb
Using sys-firewall as UpdateVM for Dom0
Downloading packages. This may take a while...
Cache directory "/var/lib/qubes/dom0-updates/var/cache/dnf" does not exist. Nothing to clean.
Updating and loading repositories:
 Fedora 37 - x86_64 ???% |   0.0   B???% |   0.0   B
 Qubes Host Repository (updates) ???% |   0.0   BA
 Fedora 37 - x86_64 100% |   4.0 KiB
 Qubes Host Repository (updates) ???% |   0.0   BA
 Fedora 37 - x86_64 100% |   4.0 KiB
 Qubes Host Repository (updates) 100% |   3.4 KiBA
 Fedora 37 - x86_64 100% |   4.0 KiB
 SecureDrop Workstation Keyring (staging) ???% |   0.0   B
 Qubes Host Repository (updates) 100% |   3.4 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Keyring (staging) ???% |   0.0   B
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% | 535.4 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64  12% |   4.7 MiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64  20% |   7.7 MiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates)   0% |   5.3 KiB
 Fedora 37 - x86_64  26% |   9.8 MiB
 Fedora 37 - x86_64 - Updates 100% |  12.2 MiB_[3A
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates)  15% | 101.3 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 Qubes Host Repository (updates)  15% | 101.3 KiB
 Qubes Host Repository (updates) 100% |   1.3 MiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 SecureDrop Workstation Keyring (staging) 100% |   1.5 KiB
 SecureDrop Workstation Keyring (staging) 100% |   3.4 KiB
 Fedora 37 - x86_64 100% |  34.6 MiB
_[?25hRepositories loaded.
_[1mPackage                                _[0m _[1mArch  _[0m _[1mVersion                _[0m _[1mReposit_[0m _[1m     Size_[0m
Installing:
_[32m securedrop-workstation-keyring-staging_[0m noarch 0.3.0-1.fc37            secured   3.3 KiB

Transaction Summary:
 Installing:         1 package

Total size of inbound packages is 16 KiB. Need to download 16 KiB.
After this operation, 3 KiB extra will be used (install 3 KiB, remove 0 B).
The operation will only download packages for the transaction.
_[?25l[1/1] securedrop-workstation-keyring-staging-0:0.3.0-1.fc37.noarch   0% |   0.0   B

[1/1] securedrop-workstation-keyring-staging-0:0.3.0-1.fc37.noarch   0% |   0.0   B

[1/1] securedrop-workstation-keyring-staging-0:0.3.0-1.fc37.noarch 100% |  15.5 KiB

[1/1] Total 100% |  15.5 KiB
Complete!
_[?25hQubes OS Repository      [===                 ] ---  B/s |   0  B     --:-Qubes OS Repository for Dom0                    2.9 MB/s | 3.0 kB     00:00    
Qubes OS Repository for Dom0                    260 kB/s | 1.1 kB     00:00    
Dependencies resolved.
================================================================================
 Package                            Arch   Version      Repository         Size
================================================================================
Installing:
 securedrop-workstation-keyring-staging
                                    noarch 0.3.0-1.fc37 qubes-dom0-cached  11 k

Transaction Summary
================================================================================
Install  1 Package

Total size: 11 k
Installed size: 3.3 k
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Installing       : securedrop-workstation-keyring-staging-0.3.0-1.fc37.   1/1 
  Running scriptlet: securedrop-workstation-keyring-staging-0.3.0-1.fc37.   1/1 
Running timer as unit: run-re067840ecef74bacb069c693dbfc7364.timer
Will run service as unit: run-re067840ecef74bacb069c693dbfc7364.service

  Verifying        : securedrop-workstation-keyring-staging-0.3.0-1.fc37.   1/1 

Installed:
  securedrop-workstation-keyring-staging-0.3.0-1.fc37.noarch                    

Complete!
make assert-keyring-staging
make[1]: Entering directory '/home/user/securedrop-workstation'
make[1]: Leaving directory '/home/user/securedrop-workstation'
make install-rpm RPM_INSTALL_STRATEGY=staging
make[1]: Entering directory '/home/user/securedrop-workstation'
# All environments depend on the prod keyring package
Installing prod keyring package
package securedrop-workstation-keyring is not installed
Using sys-firewall as UpdateVM for Dom0
Downloading packages. This may take a while...
Removed 24 files, 13 directories (total of 102 MiB). 0 errors occurred.
Updating and loading repositories:
 Qubes Host Repository (updates) ???% |   0.0   B  B
 Qubes Host Repository (updates) ???% |   0.0   BA
 Qubes Host Repository (updates) 100% |   3.4 KiBA
 Fedora 37 - x86_64 ???% |   0.0   B   4.0 KiB_[1A
 Qubes Host Repository (updates) 100% |   3.4 KiB
 Fedora 37 - x86_64 ???% |   0.0   B   4.0 KiB_[2A
 Qubes Host Repository (updates) 100% |   3.4 KiB
 Fedora 37 - x86_64 ???% |   0.0   B   4.0 KiB_[2A
 Qubes Host Repository (updates) 100% |   3.4 KiB
 Fedora 37 - x86_64 100% |   4.0 KiB   4.0 KiB_[2A
 Qubes Host Repository (updates) 100% |   3.4 KiB
 Fedora 37 - x86_64 100% |   4.0 KiB   4.0 KiB_[2A
 SecureDrop Workstation Qubes dom0 repo ???% |   0.0   B
 Qubes Host Repository (updates) 100% |   3.4 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB  12.1 KiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB  12.1 KiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB  87.2 KiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB 629.7 KiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB   2.2 MiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   5.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB   5.7 MiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates)  12% |  85.3 KiB
 Fedora 37 - x86_64 100% |  10.2 KiB   9.0 MiB_[3A
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates)  41% | 277.3 KiB
 Fedora 37 - x86_64 - Updates 100% |  11.9 MiB_[3A
 Fedora 37 - x86_64 100% |  10.2 KiB
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB
 Qubes Host Repository (updates) 100% |   1.1 MiB_[2A
 Fedora 37 - x86_64   1% |  10.2 KiB
 Fedora 37 - x86_64   1% |  10.2 KiBepo 100% |   1.5 KiB_[1A
 Fedora 37 - x86_64  27% | 630.6 KiBepo 100% |   1.5 KiB_[1A
 Fedora 37 - x86_64  10% |   3.9 MiBepo 100% |   1.5 KiB_[1A
 SecureDrop Workstation Qubes dom0 repo 100% |   3.4 KiB_[1A
 Fedora 37 - x86_64 100% |  33.6 MiB
_[?25hRepositories loaded.
_[1mPackage                        _[0m _[1mArch  _[0m _[1mVersion                _[0m _[1mReposit_[0m _[1m     Size_[0m
Installing:
_[32m securedrop-workstation-keyring_[0m noarch 0.3.0-1.fc37            secured   5.7 KiB

Transaction Summary:
 Installing:         1 package

Total size of inbound packages is 17 KiB. Need to download 17 KiB.
After this operation, 6 KiB extra will be used (install 6 KiB, remove 0 B).
The operation will only download packages for the transaction.
_[?25l[1/1] securedrop-workstation-keyring-0:0.3.0-1.fc37.noarch   0% |   0.0   B

[1/1] securedrop-workstation-keyring-0:0.3.0-1.fc37.noarch   0% |   0.0   B

[1/1] securedrop-workstation-keyring-0:0.3.0-1.fc37.noarch 100% |  17.2 KiB

[1/1] Total 100% |  17.2 KiB
Complete!
_[?25hQubes OS Repository      [===                 ] ---  B/s |   0  B     --:-Qubes OS Repository for Dom0                    2.9 MB/s | 3.0 kB     00:00    
Qubes OS Repository for Dom0                    158 kB/s | 1.0 kB     00:00    
Dependencies resolved.
================================================================================
 Package                         Arch    Version       Repository          Size
================================================================================
Installing:
 securedrop-workstation-keyring  noarch  0.3.0-1.fc37  qubes-dom0-cached   13 k

Transaction Summary
================================================================================
Install  1 Package

Total size: 13 k
Installed size: 5.7 k
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Installing       : securedrop-workstation-keyring-0.3.0-1.fc37.noarch     1/1 
  Running scriptlet: securedrop-workstation-keyring-0.3.0-1.fc37.noarch     1/1 
Running timer as unit: run-rc8c6455955e1488aa8bde089105b3f97.timer
Will run service as unit: run-rc8c6455955e1488aa8bde089105b3f97.service

  Verifying        : securedrop-workstation-keyring-0.3.0-1.fc37.noarch     1/1 

Installed:
  securedrop-workstation-keyring-0.3.0-1.fc37.noarch                            

Complete!
Install published rpm
Using sys-firewall as UpdateVM for Dom0
Downloading packages. This may take a while...
Updating and loading repositories:
 Fedora 37 - x86_64 ???% |   0.0   B% |   3.4 KiB.0   B
 Fedora 37 - x86_64 - Updates ???% |   0.0   BKiB_[1A
 Fedora 37 - x86_64 100% |   4.0 KiB
 Fedora 37 - x86_64 - Updates 100% |   3.9 KiBKiB_[2A
 Fedora 37 - x86_64 100% |   4.0 KiB
 SecureDrop Workstation Qubes dom0 repo ???% |   0.0   B
 Qubes Host Repository (updates) 100% |   3.4 KiB_[3A
 Fedora 37 - x86_64 - Updates 100% |   3.9 KiB
 Fedora 37 - x86_64 100% |   4.0 KiB
 SecureDrop Workstation Qubes dom0 repo 100% |   1.5 KiB_[2A
 Fedora 37 - x86_64 - Updates 100% |   3.9 KiB
 Fedora 37 - x86_64 100% |   4.0 KiB_[1A
 Fedora 37 - x86_64 - Updates 100% |   3.9 KiB
_[?25hRepositories loaded.
_[1mPackage                            _[0m _[1mArch  _[0m _[1mVersion                _[0m _[1mReposit_[0m _[1m     Size_[0m
Installing:
_[32m securedrop-workstation-dom0-config_[0m noarch 1.6.1rc1-1.fc37         secured 299.0 KiB
Installing dependencies:
_[32m grub2-xen-pvh                     _[0m x86_64 2.12-1.fc37             qubes-d  12.0 MiB

Transaction Summary:
 Installing:         2 packages

Total size of inbound packages is 3 MiB. Need to download 3 MiB.
After this operation, 12 MiB extra will be used (install 12 MiB, remove 0 B).
The operation will only download packages for the transaction.
_[?25l[1/2] securedrop-workstation-dom0-config-0:1.6.1rc1-1.fc37.noarch   0% |   0.0   B

[1/2] securedrop-workstation-dom0-config-0:1.6.1rc1-1.fc37.noarch   0% |   0.0   B
[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64   0% |   0.0   B

[1/2] securedrop-workstation-dom0-config-0:1.6.1rc1-1.fc37.noarch  78% |  76.8 KiB
[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64   1% |  32.0 KiB

[1/2] securedrop-workstation-dom0-config-0:1.6.1rc1-1.fc37.noarch 100% |  98.3 KiB
[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64   1% |  32.0 KiB

[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64   5% | 175.9 KiB

[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64  11% | 351.9 KiB

[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64  44% |   1.4 MiB

[2/2] grub2-xen-pvh-0:2.12-1.fc37.x86_64 100% |   3.1 MiB

[2/2] Total 100% |   3.2 MiB
Complete!
_[?25hQubes OS Repository      [===                 ] ---  B/s |   0  B     --:-Qubes OS Repository for Dom0                    2.9 MB/s | 3.0 kB     00:00    
Qubes OS Repository for Dom0                    1.1 MB/s | 4.8 kB     00:00    
Dependencies resolved.
================================================================================
 Package                         Arch   Version         Repository         Size
================================================================================
Installing:
 securedrop-workstation-dom0-config
                                 noarch 1.6.1rc1-1.fc37 qubes-dom0-cached  94 k
Installing dependencies:
 grub2-xen-pvh                   x86_64 2.12-1.fc37     qubes-dom0-cached 3.1 M

Transaction Summary
================================================================================
Install  2 Packages

Total size: 3.2 M
Installed size: 12 M
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Installing       : grub2-xen-pvh-2.12-1.fc37.x86_64                       1/2 
  Installing       : securedrop-workstation-dom0-config-1.6.1rc1-1.fc37.n   2/2 
  Running scriptlet: securedrop-workstation-dom0-config-1.6.1rc1-1.fc37.n   2/2 
Created symlink /etc/systemd/system/default.target.wants/securedrop-logind-override-disable.service → /usr/lib/systemd/system/securedrop-logind-override-disable.service.
Created symlink /etc/systemd/user/default.target.wants/securedrop-user-xfce-icon-size.service → /usr/lib/systemd/user/securedrop-user-xfce-icon-size.service.
Created symlink /etc/systemd/user/default.target.wants/securedrop-user-xfce-settings.service → /usr/lib/systemd/user/securedrop-user-xfce-settings.service.
Created symlink /etc/systemd/user/default.target.wants/sdw-notify.timer → /usr/lib/systemd/user/sdw-notify.timer.

  Verifying        : grub2-xen-pvh-2.12-1.fc37.x86_64                       1/2 
  Verifying        : securedrop-workstation-dom0-config-1.6.1rc1-1.fc37.n   2/2 

Installed:
  grub2-xen-pvh-2.12-1.fc37.x86_64                                              
  securedrop-workstation-dom0-config-1.6.1rc1-1.fc37.noarch                     

Complete!
Provide instance-specific configuration and run sdw-admin --apply.
make[1]: Leaving directory '/home/user/securedrop-workstation'
make configure-env-staging
make[1]: Entering directory '/home/user/securedrop-workstation'
Configuring staging environment
./scripts/configure-environment.py --env staging
Updated config environment to 'staging'...
'config.json' -> '/usr/share/securedrop-workstation-dom0-config/config.json'
'config.json' -> '/srv/salt/securedrop_salt/config.json'
'sd-journalist.sec' -> '/usr/share/securedrop-workstation-dom0-config/sd-journalist.sec'
'sd-journalist.sec' -> '/srv/salt/securedrop_salt/sd-journalist.sec'
./files/validate_config.py
Traceback (most recent call last):
  File "/home/user/securedrop-workstation/./files/validate_config.py", line 21, in <module>
    from sdw_util.config_types import Dom0Config, ValidationError
ModuleNotFoundError: No module named 'sdw_util.config_types'
make[1]: *** [Makefile:46: configure-env-staging] Error 1
make[1]: Leaving directory '/home/user/securedrop-workstation'
make: *** [Makefile:38: staging] Error 2
```

</details>

--- a/Makefile
+++ b/Makefile
@@ -43,7 +43,7 @@ dev staging: assert-dom0 ## Installs, configures and builds a dev or staging env
 configure-env-%:
 	@echo "Configuring $* environment"
 	./scripts/configure-environment.py --env $*
-	./files/validate_config.py
+	sdw-admin --validate
 
 .PHONY: assert-keyring-%
 assert-keyring-%: ## Correct keyring pkg installed

